### PR TITLE
Added no data feedback for individuals

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- for tracks tab for individuals, now on no available data it shows no data
- for variants tab for individuals, now on no available biosamples or biosample data it shows no data

### CODE REVIEW

File: src/components/explorer/IndividualTracks.js [hunk 1]
```diff
@@ -194,11 +194,11 @@ const IndividualTracks = ({individual}) => {
 
     return (
         <>
-        {Boolean(allFoundFiles.length) && (
+        {Boolean(allFoundFiles.length) ? (
```
Comment (ppillot) [inline src/components/explorer/IndividualTracks.js]: Just an innocent question about a part of the code that was already there: why is there a creation of a Boolean object here? Is that considered a best practise for legibility?

(As a personal preference here, I might have used `> 0` had I wanted the predicate to evaluate as a boolean)

File: src/components/explorer/IndividualVariants.js [hunk 2]
```diff
@@ -55,26 +55,25 @@ const IndividualVariants = ({individual, tracksUrl}) => {
...
-            </Descriptions.Item>
-          ))}
-        </Descriptions>
+          {ids.length ? <Descriptions layout="horizontal" bordered={true} column={1} size="small">
+              {ids.map((i) => (
+                  <Descriptions.Item key={i.key} label={i.title}>
+                      <SampleVariants id={i.key}/>
+                  </Descriptions.Item>
+              ))}
+          </Descriptions> : <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} />}
```
Comment (ppillot) [inline src/components/explorer/IndividualVariants.js:76]: The `:` is a bit hard to find in the middle of this line.

**Review by ppillot — APPROVED** [review]
Changes approved. Thanks!

**Review by gsfk — APPROVED** [review]
Looks good!

**Pull request merged**